packages/vue/src/examples/pages/category/Category.vue

```diff
@@ -61,26 +61,38 @@
         </div>
         <div class="navbar__view desktop-only">
           <span>View </span>
-          <SfIcon class="navbar__view-icon" size="10px">
-            <svg viewBox="0 0 10 10">
-              <rect width="2" height="2" fill="#1D1F22" />
-              <rect y="4" width="2" height="2" fill="#1D1F22" />
-              <rect y="8" width="2" height="2" fill="#1D1F22" />
-              <rect x="4" width="2" height="2" fill="#1D1F22" />
-              <rect x="4" y="4" width="2" height="2" fill="#1D1F22" />
-              <rect x="4" y="8" width="2" height="2" fill="#1D1F22" />
-              <rect x="8" width="2" height="2" fill="#1D1F22" />
-              <rect x="8" y="4" width="2" height="2" fill="#1D1F22" />
-              <rect x="8" y="8" width="2" height="2" fill="#1D1F22" />
-            </svg>
-          </SfIcon>
-          <SfIcon class="navbar__view-icon" size="11px">
-            <svg viewBox="0 0 11 10" fill="none">
-              <rect width="11" height="2" fill="#BEBFC4" />
-              <rect y="8" width="11" height="2" fill="#BEBFC4" />
-              <rect y="4" width="7" height="2" fill="#BEBFC4" />
-            </svg>
-          </SfIcon>
+          <SfButton class="navbar__view-button" @click="isGridView = true">
+            <SfIcon
```

Comment:
what do you think about use just SfIcon and add role="button" to it?  you can add to it aria-pressed too